web/src/utils/constants.ts

```diff
@@ -23,10 +23,22 @@ export const HFI_CALC_ROUTE = '/hfi-calculator'
 export const C_HAINES_ROUTE = '/c-haines'
 export const FIRE_BEHAVIOR_CALC_ROUTE = '/fire-behaviour-calculator'
 export const FIRE_BEHAVIOUR_ADVISORY_ROUTE = '/auto-spatial-advisory'
+export const LANDING_PAGE_ROUTE = '/'
 
 // ExpandableContainer widths
 export const PARTIAL_WIDTH = 850
 export const FULL_WIDTH = 1200
 
 // Map center
 export const CENTER_OF_BC = [-123.3656, 51.4484]
+
+// Application names
+export const C_HAINES_NAME = 'C-Haines'
+export const FIRE_BEHAVIOUR_ADVISORY_NAME = 'Auto Spatial Advisory'
+export const FIRE_BEHAVIOUR_CALC_NAME = 'FBP Go'
+export const HFI_CALC_NAME = 'FireBat'
```

Comment:
FireBat and HFI Calculator are two different applications...

Reply:
FireBat is at https://psu.nrs.gov.bc.ca/fire-behaviour-calculator
HFI Calculator is at https://psu.nrs.gov.bc.ca/hfi-calculator

Reply:
Haha, yes! Connor also caught this while I blindly followed the mockups. I'm waiting on a response from Neal on going with 7 tiles and keeping FBP Go versus dropping it and sticking with 6 tiles for now.

Reply:
Added FBP Go link to Sybrand's Github page and sorted out HFI Clac and Fire Bat links I think.

Reply:
Waiting on the PO for an HFI description to replace Lorem Ipsum placeholder